docs/docs/pricing/README.md

```diff
@@ -31,6 +31,18 @@ Free Tier accounts have a [daily credit limit](/limits#daily-credits-limit) and
 
 Upgrading to a [paid tier](https://pipedream.com/pricing) will increase the number of available active workflows and connected accounts.
 
+#### Compute Time Limits
```

Comment:
```suggestion
#### Compute time limits
```

Reply:
Changed "Time Limits" to "time limits" to follow sentence case for headings (Google's Developer docs style guide)